content/rancher/v2.x/en/installation/ha/rancheronRKE/_index.md

```diff
@@ -0,0 +1,242 @@
+
+Hardware requirements scale based on the size of your Rancher deployment. Provision each individual node according to the requirements.
+
+| Deployment Size | Clusters | Nodes | vCPUs | RAM |
+| ---- | ---- |  ---- | ---- | ---- | 
+| Small | Up to 10  | Up to 50  | 2 | 4GB |
+| Medium | Up to 100 | Up to 500 | 8 | 32GB |
+| Large | Over 100  | Over 500 | Contact Rancher |
+
+Software Requirements Docker
+
+- 1.12.6
+- 1.13.1
+- 17.03.02
+
+Note:
+If you are using RancherOS, make sure you switch the Docker engine to a supported version using sudo ros engine switch docker-17.03.2-ce
+Supported Versions
+
+Port Requirements on your Linux hosts.
+
+| Protocol | Port range | Purpose |
+| ---- | ---- |  ---- |
+| tcp | 22 | ssh server |
+| tcp | 80 | Rancher Server/ingress |
+| tcp | 443 | Rancher Server/ingress |
+| tcp | 6443 | kubernetes api server |
+| tcp | 2379-2380 | etcd server client api |
+| tcp | 10250-10256 | kubernetes components |
+| tcp | 30000-32767 | nodeport services |
+| udp | 8472 | canal |
+
+**Configure DNS**
+
+Choose a fully qualified domain name (FQDN) you want to use to access Rancher (something like rancher.yourdomain.com).
+
+You need to create a DNS A record, pointing to the IP address of your RKE nodes. If the DNS A record is created, you can validate if it’s setup correctly by running nslookup rancher.yourdomain.com. It should return the IP addresses of your RKE nodes like in the example below.
+
+```
+$ nslookup rancher.yourdomain.com
+Server:         your_nameserver_ip
+Address:        your_nameserver_ip#53
+
+Non-authoritative answer:
+Name:   rancher.yourdomain.com
+Address: ip_of_node1
+Name:   rancher.yourdomain.com
+Address: ip_of_node2
+Name:   rancher.yourdomain.com
+Address: ip_of_node3
+```
+
+**Using RKE**
+
+- Download RKE template
+  RKE uses a .yml manifest file to install and configure your k8s cluster. There are 2 templates to choose from, depending on the SSL certificate you want to use.
+
+  Download one of following templates:
+
+  - Using Self Signed Certificate [3-node-certificate.yml](https://github.com/rancher/rancher/blob/master/rke-templates/3-node-certificate.yml)
+  - Using Certificate Signed By Recognized CA [3-node-certificate-recognizedca.yml](https://github.com/rancher/rancher/blob/master/rke-templates/3-node-certificate-recognizedca.yml)
+  - Rename template to `rancher-cluster.yml`.
+
+- Edit RKE template
+  - Edit `rancher-cluster.yml` template 
+    - configure nodes section setting following placeholders
+      - `<IP>`: The IP address or hostname of the node.
+      - `<USER>`: The username to use to setup a SSH connection to the node. If the user is not the root user, make sure the user has access to the Docker socket. This can be tested by logging in on the node as the configured user and run docker ps.
+      - `<SSHKEY_FILE>`: The path of the SSH private key file used to authenticate to the node.
+    - configure addons section setting following placeholders
```

Comment:
Suggest to add subsections on what needs to be configured in cluster.yml based on each TLS  scenario. Otherwise it might be confusing why we are asking for Certificate Key when TLS is terminated at external LB.

Reply:
External lb was not added due to external lb templates wasn't available. 
Added external lb templates.